src/TuDa.CIMS.Web/Components/AssetList.razor

```diff
@@ -0,0 +1,195 @@
+<h3>Bestandsliste</h3>
+
+@code {
+
+// Parameter Items for asset list
+    [Parameter] public string[] Items { get; set; }
+
+    // TODO: remove the code for the initialization
+    private string[] _examples =
+    {
+        "Trockeneis", "Ethanol", "Natriumhydroxid", "Aceton", "Ammoniumchlorid", "Wasserstoffperoxid",
+        "Kaliumpermanganat", "Messzylinder", "Erlenmeyerkolben", "Reagenzglas"
+    };
+
+    protected override void OnInitialized()
+    {
+        Items = _examples;
+    }
```

Comment:
🔧 This can be removed and the property `Items` could be given a default value.
```csharp
[Parameter] public required List<string> Items { get; set; } = _examples
```